flash/image/classification/model.py

```diff
@@ -87,59 +88,57 @@ def __init__(
         learning_rate: float = 1e-3,
         multi_label: bool = False,
         serializer: Optional[Union[Serializer, Mapping[str, Serializer]]] = None,
+        training_strategy: Optional[str] = "default",
+        training_strategy_kwargs: Optional[Dict[str, Any]] = None,
     ):
-        super().__init__(
-            num_classes=num_classes,
-            model=None,
-            loss_fn=loss_fn,
-            optimizer=optimizer,
-            optimizer_kwargs=optimizer_kwargs,
-            scheduler=scheduler,
-            scheduler_kwargs=scheduler_kwargs,
-            metrics=metrics,
-            learning_rate=learning_rate,
-            multi_label=multi_label,
-            serializer=serializer or Labels(multi_label=multi_label),
-        )
 
         self.save_hyperparameters()
 
         if not backbone_kwargs:
             backbone_kwargs = {}
 
+        if not training_strategy_kwargs:
+            training_strategy_kwargs = {}
+
+        training_strategy_kwargs.update(
+            {
+                "ways": num_classes,
+            }
+        )
```

Comment:
This is a bit strange here, do we need this even when `training_strategy` is default?

Reply:
Done !